lib/generators/rolemodel/webpack/webpack_generator.rb

```diff
@@ -0,0 +1,61 @@
+module Rolemodel
+  class WebpackGenerator < Rails::Generators::Base
+    source_root File.expand_path('templates', __dir__)
+
+    def make_project_use_es_modules
+      say 'Configuring project to use ES Modules instead of CommonJS'
+      json = JSON.parse(File.read('package.json'))
+      json['type'] = 'module'
+      File.write('package.json', JSON.pretty_generate(json) + "\n")
+    end
+
+    def add_npm_packages
+      say 'Move Webpack to devDependencies'
+      run 'yarn remove webpack webpack-cli'
+
+      say 'Add NPM packages for compiling JS and CSS'
+      js_packages = %w[
+        @honeybadger-io/webpack
```

Comment:
Do we no longer have a standard `package.json` file? I'm wondering about where we bring in turbo, stimulus. Or should that be a separate generator

Reply:
I think those are still coming from the turbo-rails & stimulus-rails gems and then imported in application.js, directly or indirectly.  Though I'm admittedly less familiar with the webpack approach used here.